packages/pwa-kit-create-app/scripts/create-mobify-app-dev.js

```diff
@@ -124,7 +124,7 @@ const runGenerator = () => {
 
     const extension = process.platform === 'win32' ? '.cmd' : ''
     const npm = `npm${extension}`
-    const foundNpm = cp.spawnSync(npm, ['-v']).stdout.toString().trim()
+    const foundNpm = cp.spawnSync(npm, ['-v'], {shell: true}).stdout.toString().trim()
```

Comment:
This change doesn't look related.. what and why did we make this change?

Reply:
Do we need to mention this in a change log anywhere fixing a bug I suspect?

Reply:
Correct. It's an unrelated change. I added details to the PR description. 
After the latest v2 release, the Windows GA builds were failing on v2, and it seems like we decided not to fix it at that time. So I included the fix in this PR.

https://github.com/SalesforceCommerceCloud/pwa-kit/tree/release-2.8.x
https://github.com/SalesforceCommerceCloud/pwa-kit/actions/runs/8731228868/job/23956306359

> Extra: Fix failing GitHub Actions Windows builds
> Node.js issue: https://github.com/nodejs/node/issues/52554
> 
> The latest security fix on Node.js on Windows blocks the spawSync .cmd unless { shell: true } is used.

Good call on adding the change to the change log.